package.json

```diff
@@ -154,14 +154,14 @@
     "@types/redis": "^2.8.32",
     "@types/superagent": "^4.1.13",
     "@types/supertest": "^2.0.11",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
```

Comment:
@andrewburrowsmoj afaik that issue is fixed by this bump -- we don't have build errors now, instead, it seems to be genuine linting problems with new rules